ui/app/styles/components/tabs.scss

```diff
@@ -13,20 +13,26 @@
       &:focus {
         box-shadow: none;
       }
-      &.is-active,
-      &.is-active .tab {
+      &.active,
+      .tab {
```

Comment:
I think we still want to only target active tags here, right? 
```suggestion
      &.active .tab {
```

Reply:
Good catch. I'll amend.